src/components/SingleTile.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+
+const SingleTile = (props) => {
+
+    const [flipState, setFlipState] = useState(false);
+
+    const handleClick = (value) => {
+        if(value.matched === false) {
+            props.compareImage(value)
+            setFlipState(!flipState)
+        }
+        setTimeout(() => {
+            if(value.matched === false) {
+                props.compareImage(value)
+                setFlipState(flipState)
+            }
+        }, 3000);
+    }
+
+
+    let matchedCriteria 
+    if(props.onePhoto.match === true){
+        matchedCriteria = true
+    } else if(flipState === true){
+        matchedCriteria = true
+    } else {
+        matchedCriteria = false
+    }
+
+    console.log(props)
```

Comment:
I'm 100% guilty of this too, maybe we can delete any outstanding console.log or comments on the code in the final push to production branch and keep the other variant with console.log + comments in another branch?

Reply:
good idea!:)